quesma/quesma/router_test.go

```diff
@@ -263,3 +270,112 @@ func Test_matchedAgainstBulkBody(t *testing.T) {
 		})
 	}
 }
+
+const testIndexName = "indexName"
+
+func TestConfigureRouter(t *testing.T) {
+	cfg := &config.QuesmaConfiguration{
+		IndexConfig: map[string]config.IndexConfiguration{
+			testIndexName: {
+				Name: testIndexName,
+			},
+		},
+	}
+	tr := TestTableResolver{}
+	testRouter := ConfigureRouter(cfg, schema.NewSchemaRegistry(fixedTableProvider{}, cfg, clickhouse.SchemaTypeAdapter{}), &clickhouse.LogManager{}, &ingest.IngestProcessor{}, &ui.QuesmaManagementConsole{}, telemetry.NewPhoneHomeAgent(cfg, nil, ""), &QueryRunner{}, tr)
+
+	tests := []struct {
+		path                string
+		method              string
+		shouldReturnHandler bool
+	}{
+		// Routes explicitly registered in the router code
+		{routes.ClusterHealthPath, "GET", true},
```

Comment:
I'm wondering if we could get these paths from router without explicitly listing them here.

Reply:
I thought about that, but then the nature of this failure made me think that we'd be safer this way 😉